src/components/Login/LoginForm/index.js

```diff
@@ -1,24 +1,42 @@
-import React, { useMemo } from 'react';
+/* eslint-disable import/no-unresolved */
+import React, { useMemo, useEffect } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import styles from './loginForm.module.scss';
 import LoginCaption from './LoginCaption';
 import LoginButton from './LoginButton';
 import Input from '../../shared/Input';
 import { getPassword, getUsername } from '../../../store/Login/selectors';
-import { changePasswordValue, changeUsernameValue, formSubmit } from '../../../store/Login/actions';
+import { changePasswordValue, changeUsernameValue } from '../../../store/Login/actions';
 import AlertMessage from '../../shared/AlertMessage';
 import { preventDefaultHandler } from '../../../utils';
+import { login, redirectToHomeIfIsAuthorized } from '../../../store/login/actions';
+import { getErrorMessage, isError } from '../../../store/login/selectors';
 
-const ERROR_MESSAGE = 'Wrong username or password. Please, try again';
 
 const LoginForm = ({
-  username, password, changePassword, changeUsername, submitForm
+  username,
+  password,
+  changePassword,
+  changeUsername,
+  submitForm,
+  isError,
+  errorMessage,
+  redirectToHomeIfIsAuthorized
 }) => {
   const formHandler = useMemo(
     () => preventDefaultHandler(submitForm),
     [submitForm]
   );
+  const alertClasses = useMemo(() => [
+    styles.error,
+    isError ? styles.visible : styles.hidden
+  ], [isError]);
+
+  useEffect(() => {
+    redirectToHomeIfIsAuthorized();
```

Comment:
Won't it be better to create some NotLoggedRoute for such cases?

Reply:
+